pkg/util/callback.go

```diff
@@ -39,6 +39,10 @@ func (c *callbackInvoker) ExecuteCallback(callbackURL string, success bool, mess
 		"success": success,
 	}
 
+	for name, value := range labels {
+		d[name] = value
+	}
```

Comment:
@csrwng flat json is more easy to parse ;-) but yeah, I can add this into a labels field